CatProducer/scripts/catGetDatasetInfo

```diff
@@ -138,7 +138,7 @@ for d in ds:
     if len(dataSetName) < 4: continue
 
     pdName, sdName = dataSetName[1], dataSetName[2]
-    if [int(x) for x in era[1:].split('-')] >= [8,0,8]:
+    if [int(x) for x in era[1:].split('_')] >= [9,1]:
```

Comment:
have to be kept as is.

Reply:
T.J. wanted to change production tag, like 'V9_1'. https://docs.google.com/spreadsheets/d/1rWM3AlFKO8IJVaeoQkWZYWwSvicQ1QCXYSzH74QyZqE/edit#gid=1986226361

Reply:
Yes, it seems working fine.